templates/etcd.conf.yaml.j2

```diff
@@ -0,0 +1,9 @@
+data-dir: {{ etcd_data_dir }}
```

Comment:
I think we need separate variable for etcd host name. But in defaults it could have a value like this:
`etcd_node_name: {{ inventory_hostname }}`

I mean it's good to have a default value as well as give to user possibilities to rewrite it

Reply:
`name` is a standard field that is used in a config file.
https://etcd.io/docs/v3.4/op-guide/configuration/#--name